src/main/scala/nl/knaw/dans/easy/pid/ServiceStarter.scala

```diff
@@ -32,18 +32,21 @@ class ServiceStarter extends Daemon with DebugEnhancedLogging {
 
   override def start(): Unit = {
     logger.info("Starting service...")
-    service.start()
+    app.init()
```

Comment:
we not only need to start the server, but also initialize the database connection here. Below we also need to close it again in `destroy`.